src/commands/profile/profile.ts

```diff
@@ -54,12 +54,25 @@ export class ProfileCommand extends SubCommandPluginCommand {
           // iterate through each of the configurations, prettyProfileDetails making the configuration more readable
           // as opposed to snake case
           // need to cast val to string since addField does not take in numbers
-          profileDisplay.addField(prettyProfileDetails[key as keyof typeof prettyProfileDetails], val.toString());
+          if (key === 'about_me') {
```

Comment:
hmm i wonder if enum[WORD] could replace 'word'.
or, const notInLine = ['about_me', etc.] (mayhaps in components)
better practice idk?